.clang-format

```diff
@@ -59,16 +59,16 @@ ForEachMacros:
   - foreach
   - Q_FOREACH
   - BOOST_FOREACH
-IncludeBlocks:   Preserve
+IncludeBlocks:   Regroup
```

Comment:
Nah, this needs to be preserve in order to separate out problematic headers.

Reply:
Not sure about that, in my opinion we should try to not depend on include order at all. At the moment it works, are you sure that should be changed back?

Reply:
I agree we should try not to and none of the headers we write will be, but some really really dumb headers exist that are not self-sufficient.